org.eclipse.winery.topologymodeler.ui/lib/package.json

```diff
@@ -0,0 +1,49 @@
+{
+    "name": "@winery/topologymodeler",
+    "version": "0.5.4",
+    "scripts": {
+        "ng": "ng",
+        "start": "ng serve",
+        "build": "ng build",
+        "test": "ng test",
+        "lint": "ng lint",
+        "e2e": "ng e2e",
+        "lib-readme": "copy ..\\src\\doc\\lib\\README.md dist",
+        "lib-readme-mac": "cp ../src/doc/lib/README.md dist",
+        "packagr-mac": "ng-packagr -p ng-package.json && npm run lib-readme-mac",
+        "packagr": "ng-packagr -p ng-package.json && npm run lib-readme"
+    },
+    "private": false,
+    "peerDependencies": {
+        "@angular-redux/store": "6.5.7",
+        "@angular/animations": "6.0.4",
+        "@angular/cdk": "6.2.1",
+        "@angular/common": "6.0.4",
```

Comment:
Similar as mentioned above - please ensure that you use the same versions as in the other projects.

Reply:
done